kibitzr/app.py

```diff
@@ -137,13 +137,26 @@ def schedule_checks(checkers):
         schedule.clear()
         for checker in checkers:
             conf = checker.conf
-            period = conf["period"]
-            logger.info(
-                "Scheduling checks for %r every %r seconds",
-                conf["name"],
-                period,
-            )
-            schedule.every(period).seconds.do(checker.check)
+            for s in conf['schedule']:
+                job = getattr(schedule.every(s['interval']), s['unit'])
+                if s['at'] is not None:
+                    logger.info(
```

Comment:
Logger call will fail if one of the keys is missing.

Reply:
Ah, I see that keys are guaranteed to exist by conf.py

Reply:
Yes! But'garanteed' is a big word, maybe I shoud use s.get('interval',"undefined") or something like that?

Reply:
I think it's a good place for a little `namedtuple('ScheduleParameters', 'at interval unit')`. That way accessing attributes won't feel dangerous.

Reply:
will do.